### PR TITLE
Database implementation Z/D

### PR DESCRIPTION
**Pull request by Zvelebilm**
GLHF with SS

### CODE REVIEW

File: src/main/java/org/openskyt/skytale/SkytaleApplication.java
Comment (BariekR) [inline src/main/java/org/openskyt/skytale/SkytaleApplication.java:1]: Please don't use @Autowired on field. Use dependency injection through constructor instead

File: src/main/java/org/openskyt/skytale/models/Chatroom.java
Comment (BariekR) [inline src/main/java/org/openskyt/skytale/models/Chatroom.java:1]: @JoinTable is not necessary.
Chatroom constructor could be placed not in-between fields.

File: src/main/java/org/openskyt/skytale/security/SecurityConfig.java
Comment (BariekR) [inline src/main/java/org/openskyt/skytale/security/SecurityConfig.java:1]: We will have to clean this up tomorrow.

File: src/main/resources/application.properties
Comment (BariekR) [inline src/main/resources/application.properties:1]: Not sure if "security: basic: enabled: ... " is still necessary.

**Review by BariekR — APPROVED** [review]
Approved with comments.

**Review by PavelSlimBilek — APPROVED** [review]
Looking good

**Review by Lochneska — APPROVED** [review]
👍

**Review by klusi6 — APPROVED** [review]

**Pull request merged**